vm/runtime/src/lib.rs

```diff
@@ -146,6 +142,8 @@ pub trait Syscalls {
             .verify(plaintext, signer)
             .map_err(|e| ActorError::new(ExitCode::ErrPlaceholder, e))
     }
+    /// Verifies signatures of a block header
+    fn verify_block_signature(&self, bh: &BlockHeader) -> Result<(), ActorError>;
```

Comment:
This should not be added to the syscalls trait (Should be an implementation of just the specific implementation that uses it